api-server/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "engines": {
     "node": ">=16",
-    "npm": ">=8"
+    "pnpm": ">=7.20.0"
```

Comment:
Does is have to be this specific?

Reply:
Nope, I just failed at understanding semver (Shaun, please forgive me). '7' is all we need here, so we don't accidentally bump major versions.